sonic-utilities-tests/aclshow_test.py

```diff
@@ -0,0 +1,194 @@
+import sys
+import os
+from StringIO import StringIO
+import mock
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+sys.path.insert(0, modules_path)
+
+from imp import load_source
+load_source('aclshow', scripts_path+'/aclshow')
+from aclshow import *
+
+import mock_tables.dbconnector
+
+default_output = ''+ \
+"""RULE NAME     TABLE NAME      PACKETS COUNT    BYTES COUNT
+------------  ------------  ---------------  -------------
+RULE_9        DATAACL                   901            900
```

Comment:
can you explain how the tool sort the table? it does not seems to be according to the rule name and not sure if the current behavior is the expected one.

Reply:
It sorts by table name, then by counters.